x/monitoringc/types/message_create_client.go

```diff
@@ -71,9 +75,20 @@ func (msg *MsgCreateClient) ValidateBasic() error {
 	}
 
 	// check validator set hash matches consensus state
-	if !types.CheckValidatorSetHash(tmValidatorSet, tmConsensusState) {
+	if !spntypes.CheckValidatorSetHash(tmValidatorSet, tmConsensusState) {
 		return sdkerrors.Wrapf(ErrInvalidValidatorSetHash, "validator set hash doesn't match the consensus state")
 	}
 
+	// unbonding period must greater than 1 because trusting period for the IBC client is unbonding period - 1
+	// and trusting period can't be 0
+	if msg.UnbondingPeriod < spntypes.MinimalUnbondinPeriod {
+		return sdkerrors.Wrapf(ErrInvalidUnbondingPeriod, "unbonding period must be greater than 1")
+	}
+
+	// check revision height is non-null
+	if msg.RevisionHeight == 0 {
```

Comment:
Can the revision height be negative?  I mean it can because its type is `int64`.  I don't fully understand this part of the codebase yet, so I want to be sure.